qubes/mgmt.py

```diff
@@ -112,6 +112,12 @@ def __init__(self, app, src, method, dest, arg):
         #: name of the method
         self.method = method.decode('ascii')
 
+        #: callback for sending events if applicable
+        self.send_event = send_event
+
+        #: is this operation cancellable?
+        self.cancellable = False
+
```

Comment:
Added to /doc/mgmt1/ as notes. Might be a good idea to include also in (currently WIP) qubes-mgmt devel docs.